spec.html

```diff
@@ -7776,6 +7776,91 @@ <h1>Jobs and Host Operations to Enqueue Jobs</h1>
       <p>Host environments are not required to treat Jobs uniformly with respect to scheduling. For example, web browsers and Node.js treat Promise-handling Jobs as a higher priority than other work; future features may add Jobs that are not treated at such a high priority.</p>
     </emu-note>
 
+    <emu-clause id="sec-jobcallback-records">
+      <h1>JobCallback Records</h1>
+      <p>A <dfn>JobCallback</dfn> is a Record value used to store a function object and a host-defined value. Function objects that are invoked via a Job enqueued by the host may have additional host-defined context. To propagate the state, Job Abstract Closures should not capture and call function objects directly. Instead, use HostMakeJobCallback and HostCallJobCallback.</p>
```

Comment:
Either we should define the term `JobCallback Record` or the places we refer to it should just say `JobCallback`, rather than `JobCallback Record`. (In the latter case, we'd say `Return the JobCallback { ...  }` rather than `Return the JobCallback Record { ... }`, etc.)
 
There's precedent for both; e.g. [Completion](https://tc39.es/ecma262/#sec-completion-record-specification-type) is of the former type and [ExportEntry Record](https://tc39.es/ecma262/#exportentry-record) is of the latter. I think the latter is more common, so I'd be in favor of

```suggestion
      <p>A <dfn>JobCallback Record</dfn> is a Record value used to store a function object and a host-defined value. Function objects that are invoked via a Job enqueued by the host may have additional host-defined context. To propagate the state, Job Abstract Closures should not capture and call function objects directly. Instead, use HostMakeJobCallback and HostCallJobCallback.</p>
```